tox.ini

```diff
@@ -14,6 +14,7 @@ basedeps = mock>=1.3.0
 deps = {[testenv]basedeps}
        django
        keyring
+       jsonpickle
```

Comment:
Will `django` users have this dependency? Should we add some extras to `setup.py` for people that want the `django` dependencies?